src/Adafruit_TinyUSB.h

```diff
@@ -36,11 +36,19 @@
 #if TUSB_OPT_DEVICE_ENABLED
 
 #include "arduino/Adafruit_USBD_Device.h"
-#include "arduino/Adafruit_USBD_CDC.h"
+#if CFG_TUD_CDC
+  #include "arduino/Adafruit_USBD_CDC.h"
+#endif
 
-#include "arduino/hid/Adafruit_USBD_HID.h"
-#include "arduino/midi/Adafruit_USBD_MIDI.h"
-#include "arduino/msc/Adafruit_USBD_MSC.h"
+#if CFG_TUD_HID
+  #include "arduino/hid/Adafruit_USBD_HID.h"
+#endif
+#if CFG_TUD_MIDI
+  #include "arduino/midi/Adafruit_USBD_MIDI.h"
+#endif
+#if CFG_TUD_MSC
+  #include "arduino/msc/Adafruit_USBD_MSC.h"
+#endif
 #include "arduino/webusb/Adafruit_USBD_WebUSB.h"
```

Comment:
could you wrap webusb within the `CFG_TUD_VENDOR` as well for consistency

Reply:
Ok.

Done in https://github.com/adafruit/Adafruit_TinyUSB_Arduino/pull/139/commits/92d7cc9b998850138663229db531d1b2c08e765e